src/Queue.ts

```diff
@@ -11,10 +11,13 @@ export default class Queue extends EventEmitter {
   _maxConcurrent: number;
   _maxQueue: number = Infinity;
   _queue: PromiseQueue;
+  // @ts-ignore
```

Comment:
Instead of ignoring, you could change the typing of the constructor params to:

```ts
  constructor(jobTimeout: number | undefined, options: QueueOptions = { concurrency: 1 }) {
```